basil.js

```diff
@@ -1494,44 +1494,56 @@ pub.doc = function(doc) {
 /**
  * Sets the size of the current document, if arguments are given.
  * If only one argument is given, both the width and the height are set to this value.
+ * Alternatively a string can be given as the only argument to apply an existing page size preset ("A4", "Letter" etc.).
```

Comment:
(Sorry, for the late response, I am on holiday now 😎 )

No, we cannot list them in the docs, as every user can define his own presets with own names (and probably there are a lot of different presets over different versions). So, I would just leave them out and I just put "A4" and "Letter" in there as examples, so people understand right away, what we are talking about.